### ISSUE TITLE
【typescript】 Value of type 'MittStatic' is not callable. Did you mean to include 'new'?

### ISSUE THREAD

**Issue by wangduanduan** [developit/mitt#92]
i use mitt in typescript, and get an error tip

```ts
import mitt from 'mitt'
const emitter: mitt.Emitter = mitt()
```

then i get 

```
Value of type 'MittStatic' is not callable. Did you mean to include 'new'?ts(2348)
```

how to fix it, just add new ? 

thank you

**Comment by developit** [comment]
you can add `new`, though doing so has no effect since `mitt()` is a function.

The TypeScript definition here needs to be changed to allow either, if such a thing is even possible in TypeScript.

**Comment by wangduanduan** [comment]
ok, thank u

**Comment by kcharaa** [comment]
What is the solution here, adding `new` does not work since `mitt` is not a constructor!
Thanks!

**Comment by kcharaa** [comment]
Hi, when I used below import format, it works now:
```
import * as mitt from 'mitt';
let emitter: mitt.Emitter = new mitt();
```
Thanks!

**Comment by developit** [comment]
This is fixed in 1.2.0.
